site/en/blog/chromium-chronicle-21/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "The Chromium Chronicle #21: Chrome OS E2E UI Automation"
```

Comment:
Got some feedback from a reviewer that "E2E" might be ambiguous to some readers. Could we please change this instance in the title to "End-to-end"?

Reply:
Done!